examples/advanced/Tutorial3/MQ/README.md

```diff
@@ -3,3 +3,27 @@
 This part of Tutorial3 demonstrates how to transport data of Tutorial3 (FairTestDetector) via FairMQ.
 
 - \subpage ex_tutorial3_mq_dataformat
+
+## Data Format {#ex_tutorial3_mq_dataformat}
```

Comment:
Please remove this line:
```
- \subpage ex_tutorial3_mq_dataformat
```

It links to a full file with that "tag". As this file is now contained in the same file as subsection, it's not needed any longer.

If there are not other references to ` {#ex_tutorial3_mq_dataformat}`, you can remove that as well.